ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/ObservedCveModeSelect.tsx

```diff
@@ -67,15 +74,15 @@ function ObservedCveModeSelect({
             <SelectList style={{ width }}>
                 <SelectOption
                     value={observedCveModeValues[0]}
-                    description={getViewStateDescription('OBSERVED', 'WITH_CVES')}
+                    description={getViewStateDescription('OBSERVED', true)}
                 >
-                    {getViewStateTitle('OBSERVED', 'WITH_CVES')}
+                    {getViewStateTitle('OBSERVED', true)}
                 </SelectOption>
                 <SelectOption
                     value={observedCveModeValues[1]}
-                    description={getViewStateDescription('OBSERVED', 'WITHOUT_CVES')}
+                    description={getViewStateDescription('OBSERVED', false)}
                 >
-                    {getViewStateTitle('OBSERVED', 'WITHOUT_CVES')}
+                    {getViewStateTitle('OBSERVED', false)}
```

Comment:
What did you feel are pro and con for type of second argument?

You would laugh at the tug of war in my mind, with eyes 80% on green at right and 20% on red at left:
* `'OBSERVED'` and `false` means false positive or deferred?
* And then, not with CVEs

Here is a descriptive alternative than might remove redundancy and risk of inconsistency:

```tsx
observedCveModeValues.map((observedCveModeValue) => (
    <SelectOption
        key={observedCveModeValue}
        value={observedCveModeValue}
        description={getViewStateDescription('OBSERVED', observedCveModeValue)}
    >
        {getViewStateTitle('OBSERVED', observedCveModeValue)}
    </SelectOption>
)
```

Reply:
Hmm I'm not sure I follow.

I was considering removing the second parameter altogether, and having these helper functions specifically to handle the `vulnerabilityState`, regardless of whether or not the "WITH" or "WITHOUT" CVE mode is selected. Or more accurately, navigated to, in the near future. The alternative text for "WITHOUT CVES" would be delegated to the caller, and these utils would be more single responsibility?

Reply:
This selection becomes obsolete correct? If `observedCveModeValues` does too, then comment is not necessarily relevant moving forward. What I think that I noticed is that the two options could theoretically have been rendered consistently and declaratively via `map` method on the array. Instead of
* inline index `[0] and `[1]`
* that must match inline string (before change) or boolean (after change)

Reply:
Got it, yes that makes sense as a good alternative to `[0]` and `[1]`.

> This selection becomes obsolete correct?

I was actually just going to comment this as well, with future updates (and the removal of the feature flag) this entire file should be delete-able.